modules/helfi_toc/helfi_toc.libraries.yml

```diff
@@ -5,4 +5,4 @@ table_of_contents:
   dependencies:
     - core/jquery
```

Comment:
Is this actually needed anymore? At least on the tableOfContents file the $ was removed so maybe not?

Reply:
Good catch. Will remove it.